test/src/unit-capi-config.cc

```diff
@@ -226,6 +226,7 @@ void check_save_to_file() {
   ss << "config.logging_level 0\n";
 #endif
   ss << "filestore.buffer_size 104857600\n";
+  ss << "rest.curl.buffersize 524288\n";
```

Comment:
NIT: I'd suggest `buffer_size` rather than `buffersize` as a suffix, for conformity with the rest of existing `*buffer_size` config params

Reply:
Updated